packages/content-management/table_list_view_table/src/table_list_view_table.tsx

```diff
@@ -849,12 +848,10 @@ function TableListViewTableComp<T extends UserContentCommonSchema>({
         });
       }
 
-      if (data.page || !urlStateEnabled) {
```

Comment:
What is the reason to remove the condition?

Reply:
So now I want to always call dispatch directly, because it is the source of state now even when url is enabled. 

previosly it was called only if there is a `page` change or if url state was disabled